install/eclipse.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+# Installer for JetBrains Toolbox
+# This will allow the installation, upgrade and management of other JetBrains tools
+# Running the tool itself will automatically add an icon to the launcher
+
+TMPDIR=~/tools/tmp
+OUTDIR=~/tools/eclipse
+CHECKSUM="7cd7c5a020bb4a8644fdaa1b4e3b1b9daab34edd17677e28b2cdd7934f2a6432320fbba241e4fb91349ae056405721522e4bfe29970a3934d49a3e8385558691"
+ICONFILE=~/.local/share/applications/eclipse.desktop
+
+if [ ! -x "$OUTDIR/java-neon" ]; then
+
+    mkdir -p $OUTDIR
+    mkdir -p $TMPDIR
+
+    curl -L http://www.mirrorservice.org/sites/download.eclipse.org/eclipseMirror/technology/epp/downloads/release/neon/2/eclipse-java-neon-2-linux-gtk-x86_64.tar.gz -o $TMPDIR/eclipse.tar.gz
+    if [ "`sha512sum $TMPDIR/eclipse.tar.gz`" == "$CHECKSUM  $TMPDIR/eclipse.tar.gz" ]; then
```

Comment:
Some bits from https://google.github.io/styleguide/shell.xml - `$(subshell command)` is preferred to `s for readability